mitmproxy/addons/__init__.py

```diff
@@ -21,7 +23,13 @@
 from mitmproxy.addons import stickycookie
 from mitmproxy.addons import streambodies
 from mitmproxy.addons import save
+from mitmproxy.addons import tlsconfig
 from mitmproxy.addons import upstream_auth
+from mitmproxy.utils import compat
+
+if compat.new_proxy_core:  # pragma: no cover
```

Comment:
Is the still needed with the updated migration plan?

Reply:
No, we can kick this all out once merged. :)